packages/file_selector/file_selector_web/example/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: file_selector_web_integration_tests
+publish_to: none
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  build_runner: ^1.10.0
+  file_selector_web:
+    path: ../
+  flutter_driver:
+    sdk: flutter
+  flutter_test:
+    sdk: flutter
+  integration_test:
+    sdk: flutter
+
+environment:
+  sdk: ">=2.12.0-0 <3.0.0"
```

Comment:
We're using 2.12.0-259.9.beta for all the stable ranges.

Reply:
Arrrrghghgh